backend/grant/task/jobs.py

```diff
@@ -0,0 +1,16 @@
+from grant.extensions import db
+
+
```

Comment:
When we start implementing jobs, it might be good to come up with a common interface for jobs, something like:
```
proposal_reminder.create(...)
proposal_reminder.process(task...)
proposal_reminder.check_blob(blob)
```

Reply:
Totally, this makes a lot of sense. We'll have one class for each type of job.